packages/oauth/src/index.spec.js

```diff
@@ -398,7 +398,7 @@ describe('OAuth', async () => {
     });
 
     it('should call the failure callback due to missing state query parameter on the URL', async () => {
-      const req = { url: 'someUrl' };
+      const req = { url: 'http://someUrl.com' };
```

Comment:
Yes, good idea. In fact, this change is related to another change (see [my related review comment here](https://github.com/slackapi/node-slack-sdk/pull/1325#discussion_r705565947)). Because of the removal of the use of `parseUrl` (which is deprecated) and replacing it with `new URL()`, if the protocol and/or TLD is not provided in the URL string, it will cause an error. I wonder if this has farther-reaching consequences and as a result need to add more tests for this? For example, what if someone has custom domain mappings in their e.g. `hosts` file, to map e.g. `someUrl` to `127.0.0.1`. In the current implementation, perhaps it is perfectly OK to use a technically malformed URL (no protocol and/or TLD), whereas due to this change, that would not be acceptable.

What do you think?

Reply:
Thanks for sharing the context. My comment was not clear enough but I wanted to suggest `http://example.com` or `https://example.com` instead. If you click the URL in your editor, you will be navigated to the URL. Some people may unintentionally access the URLs. 

Although I don't have any thoughts on the domain itself, I think that we can always consider using the safest domains / URLs in test data. I'm not stick with example.com at all but it's commonly used. For this reason, it should be good enough.